packages/examples/compose-files-only/package.json

```
@@ -15,6 +15,8 @@
   "devDependencies": {
     "@types/ioredis": "^4.0.13",
     "@types/pg": "^7.11.1",
-    "@types/sequelize": "^4.28.4"
+    "@types/sequelize": "^4.28.4",
+    "dockest": "1.0.0",
```

```suggestion
    "dockest": "*",
```
To align with the other examples